modules/nw-networkpolicy-about.adoc

```diff
@@ -47,6 +47,8 @@ To make a project allow only connections from the {product-title} Ingress Contro
 [IMPORTANT]
 ====
 For the OVN-Kubernetes network provider plug-in, when the Ingress Controller is configured to use the `HostNetwork` endpoint publishing strategy, there is no supported way to apply network policy so that ingress traffic is allowed and all other traffic is denied.
+
+For bare metal OVN-Kubernetes installs, the Ingress Controller pods default to using the `HostNetwork` endpoint publishing strategy. There is no way to track host-networked pods within OVN. And there is no way to define any kind of network policy, which includes allow all ingress traffic with network policy on bare metal clusters.
```

Comment:
@bmcelvee 

I'm referring to this as the _OVN-Kubernetes cluster network provider_ elsewhere.

> If your cluster uses the OVN-Kubernetes cluster network provider, ...

Reply:
Thanks! I'll update both references.